go/tasks/plugins/presto/config/config.go

```diff
@@ -52,6 +57,10 @@ var (
 			Workers:      15,
 			LruCacheSize: 10000,
 		},
+		RateLimiterConfig: RateLimiterConfig{
+			Rate:  15,
```

Comment:
What will this be used for?  Will it gate _all_ calls to Mozart?  Or just Presto?  Will it be for only creation calls?  Or both creation calls and all the status checking calls in the auto-refresh cache as well?  Can you add some comments about how best to calculate a good number for this config?

Reply:
This will be used for all calls going to Mozart only  for Presto. Sure I can add some comments.

Reply:
In terms of the values themselves, I sort of took them from the awsbatch rate limiter. I'm sure we can adjust them later based on how the Presto cluster behaves